.changeset/chubby-schools-jump.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluidframework/odsp-driver": minor
+"@fluidframework/odsp-driver-definitions": minor
+---
+
+A new API getRelaySessioInfo to surface relay session info from cache, and providers to allow access to the new API
```

Comment:
Did you mean to include the `` character? Same question in other file

Reply:
Odd, I don't see the character you're referring to in the text.

Reply:
Interesting -- 
![image](https://github.com/microsoft/FluidFramework/assets/39542938/109bf865-7a60-4f12-a2f6-ea2c281ebd28)
Let me look at some other changeset files

Reply:
But I found it in visual studio. Got rid of it!